serialize-wavefront-obj/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
this repository is under the MIT license.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE